assets/main.scss

```diff
@@ -82,20 +82,8 @@ body {
   box-shadow: 0px 0px 15px 4px rgba(0,0,0,0.16);
 }
 
-/* Team Section */
-.profile-image {
-  -webkit-border-radius: 100px;
-  border-radius: 100px;
-  width: 100%;
-  height: 100%;
-  max-width: 200px;
-  max-height: 200px;
-
-  -webkit-box-shadow: 0px 1px 1px 0px rgba(0,0,0,0.25);
-  -moz-box-shadow: 0px 1px 1px 0px rgba(0,0,0,0.25);
-  box-shadow: 0px 1px 1px 0px rgba(0,0,0,0.25);
-}
-
+<<<<<<< HEAD
+=======
```

Comment:
This shouldn't be here. Along with the ones on lines 125, 197-198, and 205
And I think any css classes that start with .profile should be able to be removed.